drag_and_drop_v2/templates/html/js_templates.html

```diff
@@ -10,20 +10,20 @@
 
 <script id="zone-input-tpl" type="text/html">
     <div class="zone-row {{ id }}">
-       <label>Text</label>
+       <label>{{i18n "Text"}}</label>
        <input type="text" class="title" value="{{ title }}" />
        <a href="#" class="remove-zone hidden">
            <div class="icon remove"></div>
        </a>
        <div class="layout">
-           <label>width</label>
+           <label>{{i18n "width"}}</label>
            <input type="text" class="size width" value="{{ width }}" />
-           <label>height</label>
+           <label>{{i18n "height"}}</label>
            <input type="text" class="size height" value="{{ height }}" />
            <br />
-           <label>x</label>
+           <label>{{i18n "x"}}</label>
```

Comment:
I think using `x` and `y` for cartesian axes is actually a standard mathematical notation [used](https://es.wikipedia.org/wiki/Coordenadas_cartesianas) [in](https://de.wikipedia.org/wiki/Kartesisches_Koordinatensystem) [every](https://ja.wikipedia.org/wiki/%E7%9B%B4%E4%BA%A4%E5%BA%A7%E6%A8%99%E7%B3%BB) [human](https://zh.wikipedia.org/wiki/%E7%AC%9B%E5%8D%A1%E5%84%BF%E5%9D%90%E6%A0%87%E7%B3%BB) [language](https://pl.wikipedia.org/wiki/Uk%C5%82ad_wsp%C3%B3%C5%82rz%C4%99dnych_kartezja%C5%84skich), so these do not need to be translated.